src/libstd/net/ip.rs

```diff
@@ -59,6 +59,48 @@ pub enum Ipv6MulticastScope {
     Global
 }
 
+impl IpAddr {
+    /// Returns true for the special 'unspecified' address (0.0.0.0 in IPv4, :: in IPv6).
+    pub fn is_unspecified(&self) -> bool {
```

Comment:
Yeah, I think reusing that issue and feature makes sense, unless @alexcrichton disagrees.